src/components/ds/button/button.tsx

```diff
@@ -0,0 +1,30 @@
+import styles from "./button.module.css";
+import { Loading } from "../../icons/loading";
+import classNames from "classnames";
+
+const Button: React.FC<{
+  handleClick?: () => void;
+  text: string;
+  icon?: JSX.Element;
+  loading?: boolean;
+  disabled?: boolean;
+  variant?: "affirm" | "decline" | "build" | "download";
```

Comment:
Let use the variants outlined in Val's design system. Then add the 3 different sizes he has;
```suggestion
  intent?: "primary" | "secondary"
  size: "small" | "medium" | "large"
```